src/test/kotlin/lotto/LottoMachineTest.kt

```diff
@@ -0,0 +1,13 @@
+package lotto
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class LottoMachineTest : StringSpec({
+
+    "돈을 받고 로또를 생성한다." {
+        val lottoMachine = LottoMachine(RandomLottoNumberGenerator())
+        val lotto = lottoMachine.generateLotto(Amount(1000))
+        lotto[0].lottoNumbers.size shouldBe Lotto.SIZE
```

Comment:
```suggestion
        lotto[0].lottoNumbers shouldHaveSize Lotto.SIZE
```
kotest에서는 size 체크를 제공하고 있습니다.

Reply:
오 저런 함수가 있군요 감사합니다!